### DISCUSSION TITLE
Name or service not known

### DISCUSSION THREAD

**Discussion by zhouhoo** [apache/doris#18574] [A - General / Q&A]
When I set up a routine load  with IP configed for broker, but get machine name not know !
`kafka log-3-FAIL, event: [thrd:GroupCoordinator]: GroupCoordinator: iZm5e472qo7wdkqmet6saxZ:9092: Failed to resolve 'iZm5e472qo7wdkqmet6saxZ:9092': Name or service not known`

my routine load  config is as below:

`FROM KAFKA
(
"kafka_broker_list" = "10.30.162.53:9092",
"kafka_topic" = "persons",
"property.kafka_default_offsets" = "OFFSET_END",
"property.group.id" = "doris_persons"
);`

my doris version is 1.2.2 with docker.

anyone has the same issue?

**Comment by LemonLiTree** [comment]
Is docker started in host mode?

**Comment by zhouhoo** [comment]
I use the official [docker-compose.yaml](https://github.com/apache/doris/tree/master/docker/runtime/docker-compose-demo/build-cluster/docker-compose/3fe_3be/docker-compose.yaml) to start dockers.  Was it in host mode?

**Reply by LemonLiTree** [reply]
Probably not, I suggest you can deploy bare metal, it's easy
